priv/gettext/da/LC_MESSAGES/default.po

```diff
@@ -582,12 +582,12 @@ msgstr "Softwareopdatering tilgængelig (%{version})"
 #, elixir-format
 #: lib/teslamate_web/live/signin_live/index.html.leex:70
 msgid "Enter the verification code generated by your authenticator app."
-msgstr ""
+msgstr "Indtast verifikationskoden genereret af din authentikator app."
 
 #, elixir-format
 #: lib/teslamate_web/live/signin_live/index.html.leex:50
 msgid "Two-Factor Authentication"
-msgstr ""
+msgstr "To-faktor authentikation"
```

Comment:
To-faktor autentifikation

Reply:
Agree. I'm not completely sure how to edit the pull request other than making a new one.